### DISCUSSION TITLE
[Enhancement] [input] 希望el-input组件的change event函数可以增加oldvalue入参，记录变更前的值

### DISCUSSION THREAD

**Discussion by 847689421** [element-plus/element-plus#17003] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `input`

## Description

希望el-input组件的change event函数可以增加oldvalue入参，记录变更前的值，因为业务场景中需要使用旧值做些验证和处理。

### Additional comments

_(empty)_

**Comment by XenoAmess** [comment]
可以先拿watch顶一顶。。。但是还是有会比较好
